PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!-- Content enclosed in HTML comments will not be rendered in the Markdown, and are intended to help guide you -->
+
+<!-- This PR template is inherited from https://github.com/opendoor-labs/.github -->
+
+## Context
+
+> Why are you making this change?
+
+## Origin
+
+> What team/individual requested this change? Please link to a JIRA ticket or other documentation if applicable. If you identified this issue yourself please write "Me" (and still link to any additional context if it exists).
```

Comment:
Leaving `Context` and `Origin` alone for the time being because of Dangerfile validation